webapp/src/main/webapp/resources/output.js

```diff
@@ -12,18 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-function unknownColumnWarning() {
-  // TODO: change from hardcoded to parameters
-  const params = {
-    numNotices: 1,
-    notices: [
-      {
-        filename: "hello.txt",
-        fieldName: "column name",
-        index: 5
-      }
-    ]
+function callCorrespondingFunction(noticesJSON) {
+  const noticeContainer = JSON.parse(noticesJSON);
+  for (var i = 0; i < noticeContainer.notices.length; i++) {
+    const notice = noticeContainer.notices[i];
+    // Notice has not been implemented, output raw json
+    if (runFunction(notice.code, notice) === -1) {
+      const unknownNotice = document.createElement('p');
+      unknownNotice.innerHTML = 'NOTICE CONTAINER NOT IMPLEMENTED - RAW JSON DATA: <br><br>' + JSON.stringify(notice);
+      document.getElementById("noticeContainer").appendChild(unknownNotice);
+    };
   }
+}
+
+function runFunction(name, arguments) {
+  const fn = window[name];
+  // check if fn is a function
+  if (typeof fn !== 'function') return -1;
+  fn.apply(window, [arguments]);
+}
+
+function unknown_column(params) {
   const template = goog.soy.renderAsElement(validator.templates.unknownColumnNotice, params);
-  document.getElementById("warning").appendChild(template);
+  document.getElementById("noticeContainer").appendChild(template);
```

Comment:
I thought we wanted one contained to warnings and another for errors.

Reply:
Ah yes, fixed.